src/app/services/postman-sms/postman-sms.service.ts

```diff
@@ -0,0 +1,161 @@
+import axios, { AxiosError } from 'axios'
+import mongoose from 'mongoose'
+import { errAsync, okAsync, ResultAsync } from 'neverthrow'
+
+import { isPhoneNumber } from '../../../../shared/utils/phone-num-validation'
+import { AdminContactOtpData, FormOtpData } from '../../../types'
+import { useMockPostmanSms } from '../../config/config'
+import { postmanSmsConfig } from '../../config/features/postman-sms.config'
+import { createLoggerWithLabel } from '../../config/logger'
+import getFormModel from '../../models/form.server.model'
+import {
+  DatabaseError,
+  MalformedParametersError,
+} from '../../modules/core/core.errors'
+import { getMongoErrorMessage } from '../../utils/handle-mongo-error'
+import MailService from '../mail/mail.service'
+
+import { InvalidNumberError, SmsSendError } from './postman-sms.errors'
+import { SmsType } from './postman-sms.types'
+import { renderVerificationSms } from './postman-sms.util'
+
+const logger = createLoggerWithLabel(module)
+const Form = getFormModel(mongoose)
+class PostmanSmsService {
+  /**
+   * Send SMS using Postman API to Member of public
+   *
+   * SMSes will be sent using govsg sender id.
+   * Messages to any member of public MUST be sent using this method.
+   */
+  sendMopSms(
+    smsData: FormOtpData | AdminContactOtpData,
+    recipient: string,
+    message: string,
+    smsType: SmsType,
+    senderIp?: string,
+  ): ResultAsync<true, SmsSendError | InvalidNumberError> {
+    const logMeta = {
+      action: 'sendMopSms',
+      recipient,
+      smsType,
+      smsData,
+      senderIp,
+    }
+
+    const body = {
+      recipient: recipient.replace('+', ''),
+      language: 'english',
+      values: { body: message },
+    }
+
+    if (useMockPostmanSms) {
```

Comment:
return result async as well for consistency?

Reply:
Is this referring to L53 instead?

`MailService.sendLocalDevMail` itself returns an asyncResult.